packages/taiko-client/bindings/encoding/protocol_config.go

```diff
@@ -19,7 +19,7 @@ var (
 		LivenessBond:          livenessBond,
 		StateRootSyncInternal: 16,
 		MaxAnchorHeightOffset: 64,
-		OntakeForkHeight:      2,
+		OntakeForkHeight:      0,
```

Comment:
Did it change to 0 in taiko-geth as well?

Reply:
Not yet, will do after this one merged into `main`

Reply:
Got it, so the CI won't succeed.